_core/smart/tutorial.md

```diff
@@ -1,211 +1,188 @@
 ---
 layout: smart
-description: "Clarity: Hello World Tutorial"
+description: Get Started Writing Smart Contracts on the Blockchain with Clarity
 permalink: /:collection/:path.html
 ---
-# Hello World
+
+# Get Started
+
+## Overview
 
 | Experience | | **Beginner**  |
 | Duration | | **15 minutes** |
 
-In this tutorial, you learn how to use Clarity, Blockstack's smart contracting language. By the end of this tutorial, you will ...
+Clarity, Blockstack's smart contracting language, is based on [LISP](https://en.wikipedia.org/wiki/Lisp_(programming_language)). Clarity is an interpreted language, and [decidable](https://en.wikipedia.org/wiki/Recursive_language). In this tutorial, you will learn how to use Clarity and how to deploy and interact with a smart contract on the Stacks 2.0 blockchain. By the end of this tutorial, you will:
```

Comment:
Removed link and just mentioned that it's based on LISP and uses the parenthesized notation; that way rather than distracting people with some side-journey about LISP, we keep things moving but plant the seed that we're using a well-documented syntax.